stories/partials/media-image.html.twig

```diff
@@ -2,7 +2,7 @@
   {% set attributes = create_attribute() %}
 {% endif %}
 
-<figure{{ attributes.addClass(additional_classes) }}>
+<figure{{ attributes.addClass(additional_classes) }} {% if aria_hidden %}{{ attributes.setAttribute('aria-hidden', true) }}{% endif %}>
```

Comment:
Fixes a11y issue in axe devtools.

The `figure` element isn't an allowed child element. This is a decorative image, so hiding solves this probem.

Supported elements for `<dl>` are: `dt, dd, groups, <script>, <template>, and <div>`.